src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -68,7 +69,10 @@ function ProviderSelectionField({
           <button
             className="va-button-link"
             type="button"
-            onClick={() => setShowProvidersList(true)}
+            onClick={() => {
+              setShowProvidersList(true);
+              recordEvent({ event: 'vaos-choose-provider-click' });
```

Comment:
We have a `GA_PREFX` constant that we try to use instead of `vaos` directly (looks like there are some others we need to clean up).

Reply:
Done